_author_instructions/01_policies.md

```diff
@@ -126,27 +121,20 @@ Additionally, you must upload a representative image as noted above.
 Article submission also involves paying a nominal charge of of $100 per submission, which covers our peer review management system as well as ongoing operation costs (web hosting, etc.).
 This is handled through Stripe Connect and can be paid via any major credit card.
 
-Additionally, you must upload a representative image as noted above.
-
-Article submission also involves paying a nominal charge of of $100 per submission, which covers our peer review management system as well as ongoing operation costs (web hosting, etc.).
```

Comment:
Good catch; not sure how this got duplicated, maybe on Michael's manual merges a couple days ago.